mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,189 @@
+{
+  "aliases": null,
+  "settings": {
+    "index.stuff": null
+  },
+  "mapping": {
+    "dynamic": false,
+    "date_detection": false,
+    "properties": {
+      "file_autocomplete": {
+        "fields": {
+          "analyzed": null,
+          "lowercase": null
+        },
+        "type": "keyword"
+      },
+      "study_id": {
+        "type": "keyword"
+      },
+      "object_id": {
+        "type": "keyword"
+      },
+      "file_type": {
+        "type": "keyword"
+      },
+      "data_type": {
+        "type": "keyword"
+      },
+      "file_access": {
+        "type": "keyword"
+      },
+      "analysis": {
+        "properties": {
+          "analysis_id": {
+            "type": "keyword"
+          },
+          "analysis_type": {
+            "type": "keyword"
+          },
+          "analysis_version": {
+            "type": "integer"
+          },
+          "experiment": {
+            "type": "object",
+            "properties": {
+              "platform": {
+                "type": "keyword"
+              },
+              "platform_model": {
+                "type": "keyword"
+              },
+              "sequencing_date": {
+                "type": "keyword"
+              },
+              "library_name": {
+                "type": "keyword"
+              },
+              "library_strategy": {
+                "type": "keyword"
+              },
+              "sequencing_center": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      "dynamic": {
```

Comment:
is there a reason to wrap those inside dynamic ?

Reply:
i assumed we wanted them grouped into a "general area" rather than flat - other than that no reason i can remove.

Reply:
If the properties under here are meant to from workflow, maybe just call it `workflow` instead of `dynamic`?

Reply:
@junjun-zhang there are more than just workflow fields that could be dynamic 
@blabadi, if its not necessary to group all dynamic fields together then i will just remove.

Reply:
I think it's still possible and good to have a dedicated `workflow` object regardless what other properties go under dynamic.

Reply:
agree - just meant to explain why i had included a group as there could be more than just the workflow object.

Reply:
i have added this.